MyBus/MyBus/MyBusMapController.swift

```diff
@@ -57,6 +57,9 @@ class MyBusMapController: UIViewController, MGLMapViewDelegate, UITableViewDeleg
         NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(MyBusMapController.offlinePackProgressDidChange(_:)), name: MGLOfflinePackProgressChangedNotification, object: nil)
         NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(MyBusMapController.offlinePackDidReceiveError(_:)), name: MGLOfflinePackProgressChangedNotification, object: nil)
         NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(MyBusMapController.offlinePackDidReceiveMaximumAllowedMapboxTiles(_:)), name: MGLOfflinePackProgressChangedNotification, object: nil)
+        
+        mapView.showsUserLocation = true
```

Comment:
Although this solution effectively shows the user's location annotation in the map, unfortunately doesn't center the map within the user's location coordinates. I guess there's a suitable method that could work for you in MyBusMapView class: **centerMapWithGPSLocation()**